src/explore-education-statistics-common/src/modules/release/components/ReleaseMetaGuidanceDataFile.tsx

```diff
@@ -64,26 +64,43 @@ const ReleaseMetaGuidanceDataFile = ({ subject, renderContent }: Props) => {
         {contentItem}
       </SummaryList>
 
-      <Details summary="Variable names and descriptions">
-        <p>Variable names and descriptions for this file are provided below:</p>
+      {variables.length && (
+        <Details
+          summary="Variable names and descriptions"
+          className="govuk-!-margin-bottom-4"
+        >
+          <p>
+            Variable names and descriptions for this file are provided below:
+          </p>
 
-        <table>
-          <thead>
-            <tr>
-              <th scope="col">Variable name</th>
-              <th scope="col">Variable description</th>
-            </tr>
-          </thead>
-          <tbody>
-            {variables.map(({ value, label }) => (
-              <tr key={value}>
-                <td className={styles.tableOverflowWrap}>{value}</td>
-                <td>{label}</td>
+          <table data-testid="Variables">
+            <thead>
+              <tr>
+                <th scope="col">Variable name</th>
+                <th scope="col">Variable description</th>
               </tr>
+            </thead>
+            <tbody>
+              {variables.map(({ value, label }) => (
+                <tr key={value}>
```

Comment:
Yeh it's not guaranteed to be unique as it can be either a filter item or an indicator, so theoretically there could be instances where these are named the same. I'm not entirely sure if the various data file validators along the way check against this?

In this context, it doesn't really matter though as we don't change the ordering. To avoid any potential question marks around duplicate key warnings in the console, I'll swap it to use the index.